index.html

```diff
@@ -697,6 +697,36 @@
   </div>
 </div>
 <!-- Images to video -->
+<!-- Transcode to FFv1.mkv -->
+<span data-toggle="modal" data-target=".create_ffv1_mkv"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Transcode your file with the FFv1 Version 3 Codec in a matroska container">Create FFv1.mkv</button></span>
+<div class="modal fade create_ffv1_mkv" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="well">
+        <h3>Create FFv1 Version 3 video in a Matroska container with framemd5 of input</h3>
+        <p><code>ffmpeg -i <i>input_file</i> -map 0 -dn -c:v ffv1 -level 3 -g 1 -slicecrc 1 -slices 16 -c:a copy <i>output_file</i>.mkv -f framemd5 -an <i>md5_output_file</i></code></p>
+        <p>This will losslessly trancode your video with the FFv1 Version 3 codec in a Matroska container. In order to verify losslessness, a framemd5 of the source video is also generated. For more information on FFv1 encoding, <a href="https://trac.ffmpeg.org/wiki/Encode/FFV1" target="_blank">try the ffmpeg wiki.</a> </p>
+        <dl>
+          <dt>ffmpeg</dt><dd>starts the command.</dd>
+          <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file.</dd>
+          <dt>-map 0</dt><dd>Map all streams that are present in the input file. This is important as ffmpeg will map only one stream of each type (video, audio, subtitles) by default to the output video.</dd>
+          <dt>-dn</dt><dd>ignore data streams (data no). The matroska container does not allow data tracks.</dd>
```

Comment:
I was tempted to use `-map 0:v` and `-map 0:a`, but some files may have subs as well. `-map 0 -dn` seems like a better workaround, but any suggestions welcome.